tests/test_creation/test_binary_combination_feature_reference.py

```diff
@@ -0,0 +1,276 @@
+import pandas as pd
+import pytest
+
+from feature_engine.creation import BinaryCombinatioByFeatureReference
+
+
+# test param variables_to_combine
+def test_error_when_param_variables_not_entered():
+    with pytest.raises(TypeError):
+        BinaryCombinatioByFeatureReference()
+
+
+def test_error_when_user_enters_one_variable():
+    with pytest.raises(KeyError):
+        BinaryCombinatioByFeatureReference(variables_to_combine=["Age"])
```

Comment:
I think we should allow 1 variable. Is this error because user wants to combine 1 variable? or because there is no reference?